recipe/meta.yaml

```diff
@@ -1,19 +1,23 @@
 {% set name = "arcticdb" %}
-{% set version = "4.0.0" %}
+{% set version = "4.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 52969d055ca622209e430f96deff33004113f0df29a0f1cb7a3538925986f529
+  sha256: 207b8951e4623bb2e87e0a3b3014ae12b7c4a696e7377b1910d9978dd1ad28eb
 
 build:
   # We skip the build on Windows because of linkage problems with libprotobuf.
   # See: https://github.com/man-group/ArcticDB/pull/449
   skip: true  # [win]
-  number: 2
+  # The build of libprotobuf is lacking the _CFRelease symbol
+  # Fixing requires to rebuild the whole stack while OSX build 
+  # for python 3.8 is not used  by anyone.
+  skip: true  # [osx and py<39]
```

Comment:
We have this on `main` so I thought I might as well bring it here too.

Reply:
Indeed, thanks.